src/components/Login/LoginForm.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import FacebookLogin from "react-facebook-login";
+import GoogleLogin from "react-google-login";
+import { Redirect } from "react-router-dom";
+
+class LoginForm extends Component {
+  render() {
+    const profileContent = this.props.isLegit ? (
+      <div>
+        <Redirect
+          to={{
+            pathname: "/"
+          }}
+        />
+      </div>
+    ) : (
+      <React.Fragment>
+        <FacebookLogin
+          appId={this.props.fbApi}
+          fields={this.props.facebookScopes}
+          onClick={this.props.loginRequested}
```

Comment:
loginRequested? It doesn't suggest me that it's onClick action. I would use sth like - loginRequest, loginOnClick, loginAction ? sth like that?

Reply:
Agreed